projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -17,13 +17,15 @@ export const settings = {
 			[]
 		);
 		const isSavingPost = useSelect( select => select( editorStore ).isSavingPost(), [] );
```

Comment:
Maybe not for this PR, but since this is starting to grow, maybe there are opportunities to simplify some things a bit?

```diff
diff --git a/projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js b/projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js
index 6fc0f5e876..9364c96b3d 100644
--- a/projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js
+++ b/projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js
@@ -12,15 +12,13 @@ export const name = 'launchpad-save-modal';
 
 export const settings = {
 	render: function LaunchpadSaveModal() {
-		const isSavingSite = useSelect(
-			select => select( editorStore ).isSavingNonPostEntityChanges(),
-			[]
-		);
-		const isSavingPost = useSelect( select => select( editorStore ).isSavingPost(), [] );
-		const isPublishingPost = useSelect( select => select( editorStore ).isPublishingPost(), [] );
-		const isCurrentPostPublished = useSelect(
-			select => select( editorStore ).isCurrentPostPublished(),
-			[]
+		const { isSavingSite, isSavingPost, isPublishingPost, isCurrentPostPublished } = useSelect(
+			selector => ( {
+				isSavingSite: selector( editorStore ).isSavingNonPostEntityChanges(),
+				isSavingPost: selector( editorStore ).isSavingPost(),
+				isPublishingPost: selector( editorStore ).isPublishingPost(),
+				isCurrentPostPublished: selector( editorStore ).isCurrentPostPublished(),
+			} )
 		);
 
 		const prevIsSavingSite = usePrevious( isSavingSite );

```

Reply:
I'll create a follow-up refactoring the launchpad save modal code 👍